src/Storage/Storage.Management/File/StorageFileBaseCmdlet.cs

```diff
@@ -16,11 +16,11 @@
 using Microsoft.Azure.Commands.ResourceManager.Common;
 using Microsoft.Azure.Management.Storage;
 using Microsoft.Azure.Management.Storage.Models;
+using Microsoft.Azure.Management.WebSites.Version2016_09_01.Models;
```

Comment:
The namespace looks a little odd. 
Why need reference this?

Reply:
Not sure why this reference is added here... As this is unused, removed it.